notebooks/Bulk Update Maps for Use in Collector or Field Maps.ipynb

```diff
@@ -0,0 +1,194 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Bulk update maps for Use in Collector / Field Maps"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook allows the user to define a search string for webmaps and set the ability to \"Use in Field Maps\" and/or \"Use in Field Maps\". It will first search for maps based on a string you define, confirm these are the maps you'd like to update, and then update each for Collector / FieldMaps use based on parameters you define. Additionally, this notebook can be imported in ArcGIS Online and used as a hosted notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter password: ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "from arcgis.gis import GIS\n",
+    "gis = GIS(\"https://arcgis.com\",\"workforce_scripts\", verify_cert=False) \n",
+    "# gis = GIS(\"home\") for Hosted or Pro Notebooks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Please set the parameters below. Example parameters have been set for this notebook.\n",
+    "\n",
+    "Note here that we include an example query which would take all maps with \"Hydrant\" in its title that are currently enabled for Collector and allow you to update them for Field Maps."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# this takes \"enabled\", \"disabled\", or \"no_change\"\n",
+    "field_maps_usage = \"enabled\"\n",
+    "collector_usage = \"no_change\"\n",
+    "webmap_search_query = \"Hydrant\"\n",
```

Comment:
I think this script is generic in setting permissions - yes I agree the primary use was to disable field maps for collectordisabled=true (or not-collector ready), so your search is a good one.  perhaps a couple of other examples could be put in, but this could be the active one?